src/fiatconnect/slice.ts

```diff
@@ -124,10 +131,13 @@ export const slice = createSlice({
   initialState,
   reducers: {
     submitFiatAccount: (state, action: PayloadAction<SubmitFiatAccountAction>) => {
-      state.sendingFiatAccount = true
+      state.sendingFiatAccountStatus = SendingFiatAccountStatus.Sending
```

Comment:
This `sendingFiatAccountStatus` field should probably be added to the REHYDRATE so that it resets when the app reloads

Reply:
ty! added